docs/decisions/0005-split-newrelic-app.rst

```diff
@@ -0,0 +1,60 @@
+Split NewRelic App
+******************
+
+Status
+======
+
+Accepted
+
+Context
+=======
+
+When using NewRelic, edx-platform as a monolithic APM app has the following issues:
+
+* Errors or other data can get lost in Overview due to the number of different types of transactions
+* Overview is not helpful for most teams that care about different parts of the app.
+
+Decision
+========
+
+We will employ a "hacked" solution to set the NewRelic app name per transaction. This is considered a reasonable "hack" by NewRelic, but NewRelic dashboards are their more general recommendation.
+
+We will define the mappings from request path to new app name in code. This mapping handler can be configured using a new setting, ``NEWRELIC_PATH_TO_APP_NAME_SUFFIX_HANDLER``.  The handler should take a request path and return None for the default mapping, or a suffix to be appended to the default app name. Providing the mapping as a setting enables quick rollback, and enables mapping overrides for the Open edX community.
```

Comment:
We should be clear that it's clear that the value of this setting is a path and not a dict.